cpp/src/community/detail/maximal_independent_moves.cuh

```diff
@@ -0,0 +1,313 @@
+
+/*
+ * Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
Is this a new file, or is this a copy/adaptation of an old file?

Reply:
This is old file, just renamed.

Reply:
Use `git mv` for just renaming a file.

Reply:
Skip reviewing this file assuming this is just renaming.